app/src/main/res/values/strings.xml

```diff
@@ -68,7 +68,10 @@
     <string name="helper_type_button_transcript">"Transcript a call!"</string>
     <string name="helper_call_overview_screen_title">"Open calls:"</string>
     <string name="seeker_phone_call_text">"Please call <br><b>%1$s</b></br>to record your shopping order."</string>
-    <string name="seeker_type_screen_title">"How would you like to make your list?"</string>
+    <string name="seeker_type_screen_title">"How would you like
+to make your list?"</string>
+    <string name="seeker_type_button_record_list">"Record my
+shopping list"</string>
```

Comment:
was this intended?

Reply:
yes, if you compare the screen designs you will see there are some intentional line breaks. The text in some of the buttons already uses intentional line breaks